tuts/create-your-first-substrate-chain/v2.0.0-rc2/interact.md

```diff
@@ -61,41 +61,72 @@ $ ./target/release/node-template --dev
 If the number after `finalized:` is increasing, your blockchain is producing new blocks and reaching
 consensus about the state they describe!
 
-## Start the Front End
+## Start the Front-End
 
-To interact with the local node we will use the Polkadot-js Apps user interface, often known as
-"Apps" for short. Despite the name, Apps will work with any Substrate-based blockchain including
-ours, not just Polkadot.
+To interact with your local node, we will use
+[the Substrate Developer Hub Front-End Template](https://github.com/substrate-developer-hub/substrate-front-end-template),
+which is a collection of UI components that have been designed with common uses cases in mind.
 
-In your web browser, navigate to
-[https://polkadot.js.org/apps](https://polkadot.js.org/apps/#/settings?rpc=ws://127.0.0.1:9944).
+To get started with the Front-End Template, clone its repository and
+[follow the simple steps to run it locally](https://github.com/substrate-developer-hub/substrate-front-end-template#using-the-template).
 
-On the `Settings` tab ensure that you are connected to a `Local Node` or `ws://127.0.0.1:9944`.
+## Interact
 
-> Some browsers, notably Firefox, will not connect to a local node from an https website. An easy
```

Comment:
Is this paragraph present somewhere else? It's good info imo.

Reply:
I don't believe that this is relevant anymore, though. This is because the tutorial instructs the user to host the front-end interface locally.

Reply:
Ok, got it.